<?xml version="1.0"?>
<launch>
	
	<group ns="Test">
		
		<!-- rosparam command="load" file="$(find my_pkg)/config/my_file.yaml" / -->
		
		<node name="QuEst" pkg="vbme_pkg" type="QuEst" output="screen" >
		</node>
		
		<param name="video_source" value="/home/codyl/Desktop/image_00/" />
		<!-- Location of the kitti data set for image publisher -->
		
		<remap from="/input/image_raw/uncompressed" to="/Test/cam1/rgb/image_color" />
		<node name="KittiPublisher1" pkg="vbme_pkg" type="image_publisher.py" output="screen">
		</node>
		
		<remap from="/input/image_raw/uncompressed" to="/Test/cam2/rgb/image_color" />
		<node name="KittiPublisher2" pkg="vbme_pkg" type="image_publisher_ahead8.py" output="screen">
		</node>
		
<!--
		<param name="video_source" value="/home/codyl/Desktop/image_00/" />
		<node name="KittiPublisher" pkg="vbme_pkg" type="image_publisher.py" output="screen">
			<remap from="/input/image_raw/uncompressed" to="/Test/cam2/rgb/image_color" />
		</node>
-->
		
<!--
		<remap from="/vbme/Transform" to="/Test/quest_msg" />
-->
<!--
		<remap from="/Test/quest_msg" to="/vbme/Transform" />
-->
		<node name="TransformPlotter" pkg="vbme_pkg" type="transform_plotter.py" output="screen" >
			<remap from="Transform" to="quest_msg" />
		</node>
		<param name="UseRansac" value="True" />
		
	</group>
</launch>
